modules/ROOT/pages/release-notes/release-notes52.adoc

```diff
@@ -25,37 +26,38 @@ The following new features and enhancements were added for the {productname} 5.2
 
 The `toolbar_location` option is used to position the toolbar and menu bar. Setting this option to `bottom` positions the toolbar and menu bar below the content area.
 
-For information on using the `toolbar_location` setting, see: link:{baseurl}/configure/editor-appearance/#toolbar_location[User interface options - `toolbar_location`].
+For information on using the `toolbar_location` setting, see: xref:configure/editor-appearance.adoc#toolbar_location[User interface options - `toolbar_location`].
```

Comment:
This xref doesn't resolve as the `toolbar_location` id doesn't exist on the relevant heading in `toolbar_location.adoc`. e.g:
```
[[toolbar_location]]
== `toolbar_location`
```

Reply:
done